app/code/Magento/Indexer/Console/Command/IndexerResetStateCommand.php

```diff
@@ -29,23 +29,29 @@ protected function configure()
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $indexers = $this->getIndexers($input);
+
+        $success = true;
```

Comment:
Hello @ihor-sviziev 
I think it would be better `$success = false` and change it on `true` only if indexer jobs have been finished successfully.
What do you think about it?

Reply:
The original idea was the following: in case of any error happens - we're returning the failure - otherwise a success.

Suppose we change the default value to false. In that case, we basically will have an issue with detecting if we had an issue or not, so we should use three options null (for the case when no information yet), true (for success), and false (for the case if any error happened).

I think the first solution is just more straightforward. Does it make sense?

Reply:
Ok.
Thank you for your clarification.
So, this pull request looks good for me ✔️